website/pages/blog/announcing-cloudquery-aws-event-based-sync-beta.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Announcing Closed Beta of CloudQuery AWS Plugin with Event-based Sync
+tag: announcement
+date: 2023/08/31
+description:
+author: michal
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+We are excited to announce the preview of an event-based sync for our [AWS Plugin](https://www.cloudquery.io/docs/plugins/sources/aws/overview)!
```

Comment:
```suggestion
We are excited to announce the preview of an event-based sync for our AWS Plugin!
```

Maybe we should not make this a link, if users don't know about the AWS plugin already this post won't really benefit them